INTMDB-299: Support Cloud Backup Export Jobs (#673)
mongodb/terraform-provider-mongodbatlas: website/docs/r/cloud_backup_snapshot_export_bucket.html.markdown

---
layout: "mongodbatlas"
page_title: "MongoDB Atlas: cloud_backup_snapshot_export_bucket"
sidebar_current: "docs-mongodbatlas-resource-cloud_backup_snapshot_export_bucket"
description: |-
    Provides a Cloud Backup Snapshot Export Bucket resource.
---

# mongodbatlas_cloud_backup_snapshot_export_bucket

> `mongodbatlas_cloud_backup_snapshot_export_bucket` esource allows you to retrieve all the buckets for the specified project. 

Reviewer: ```suggestion
`mongodbatlas_cloud_backup_snapshot_export_bucket` resource allows you to create an export snapshot buckets for the specified project. 
```

Author: done

> `mongodbatlas_cloud_backup_snapshot_export_bucket` resource allows you to retrieve all the buckets for the specified project. 

Reviewer: ```suggestion
`mongodbatlas_cloud_backup_snapshot_export_bucket` resource allows you to create an export snapshot buckets for the specified project. 
```

Author: done

-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.

## Example Usage

```terraform
resource "mongodbatlas_cloud_backup_snapshot_export_bucket" "test" {
  project_id   = "{PROJECT_ID}"
  iam_role_id = "{IAM_ROLE_ID}"
  bucket_name = "example-bucket"
  cloud_provider = "AWS"
}
```

## Argument Reference

* `project_id` - (Required) The unique identifier of the project for the Atlas cluster.

> * `iam_role_id` - (Required) Unique identifier of the role that Atlas can use to access the bucket. You must also specify the `bucketName`.

Reviewer: ```suggestion
* `iam_role_id` - (Required) Unique identifier of the role that Atlas can use to access the bucket. You must also specify the `bucket_name`.
```

> * `bucket_name` - (Required) Name of the bucket that the provided role ID is authorized to access. You must also specify the `iamRoleId`.

Reviewer: ```suggestion
* `bucket_name` - (Required) Name of the bucket that the provided role ID is authorized to access. You must also specify the `iam_role_id`.
```
